pkg/verifier/notation/certstoresbytype.go

```diff
@@ -28,6 +28,7 @@ type certStoreType string
 const (
 	CA               certStoreType = "CA"
```

Comment:
nit: is there a reason why we use `CA` in caps versus `tsa` in lower case for the new tsa store type?